routing/routing_benchmarks/bicycle_routing_tests.cpp

```diff
@@ -45,7 +45,7 @@ UNIT_CLASS_TEST(BicycleTest, Smoke)
   TestRouters(start, final);
 }
 
-UNIT_CLASS_TEST(BicycleTest, MailRuOfficeToUlSimanova5K2)
```

Comment:
It doesn't. Yesterday I tried to reproduce the problem in the second case, on map. I tapped on the Mail.ru office, then I tapped on the "Domumetro" hotel. And the bicycle router was able to construct a route between these two points. On the other hand, when I tried to construct a route between two points from the test body (exact lat-lons), the bicycle router failed. So, my point is that these text descriptions are not helpful, they're misleading. Because in any case, when you try to reproduce a problem in a failing test, you should use lat lons from the test body, not names from the test name. Please, take a look at pedestrian tests in the nearby file - all tests are named in a simple manner, on the other hand if you are interested in how these routes look like, all what you need is just take exact coords from test bodies and feed them to a geocoder.